README.md

```diff
@@ -27,3 +27,13 @@ Generate various maps as png images:
 ```
 node tools/maps.js --input ./server/1.12.2/world --bounds -256 256 -256 256 --version 1.12.2
 ```
+
+Generate a village centered at 0, 0:
+```
+node tools/genvillage.js
+```
+
+Start a Minecraft server to view the generated world/village:
+```
+node tools/server.js --seed 6708951544769826080 --version 1.12.2
```

Comment:
the seed isn't needed but we should be able to give it a world, the script would then make a copy of the world in the server directory before launching it. If the seed is left blank in the server.properties, the server should read it from the world directory.

Reply:
Oh yeah, good call.

Reply:
> the seed isn't needed but we should be able to give it a world, the script would then make a copy of the world in the server directory before launching it. If the seed is left blank in the server.properties, the server should read it from the world directory.

Wait, why would it matter for copying before viewing? Shouldn't we copy before generating?

Reply:
@Karang I added world backups before generating the village. However, I don't think making another backup before viewing the generated village is needed. Starting the server should not change the world, right?

Reply:
its not a problem of backup, its just that the input is not necessarily in the correct folder. Imagine you have multiple world you want to view, first you want to look at world 1, then world 2, you need to move them to server folder. If you don't want to copy you could make a symlink, but copy is more portable (and the world are supersmall anyway)

Reply:
Oh, okay. So like choose what world to view. Got it.